gitops/namespaces/base/networking/external-dns/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- external
+- internal
```

Comment:
Are these directories?

Reply:
It's hard to tell in the PR as I'm only having a quick glance over

Reply:
These are indeed directories and would like to know if it's necessary. In my short testing I have one externaldns deployed like:

```
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
resources:
- ../../base/networking
- ../../base/traefik/
- ../../base/traefik/traefik
- ../../base/flux-system
- ../../base/networking/external-dns/internal
```